src/ontology/chmo-odk.yaml

```diff
@@ -0,0 +1,64 @@
+## ontology metadata ##
+
+id: chmo
+title: Chemical Methods Ontology
+description: "CHMO, the chemical methods ontology, describes methods used to collect data in chemical experiments, such as mass spectrometry and electron microscopy prepare and separate material for further analysis, such as sample ionisation, chromatography, and electrophoresis synthesise materials, such as epitaxy and continuous vapour deposition It also describes the instruments used in these experiments, such as mass spectrometers and chromatography columns. It is intended to be complementary to the Ontology for Biomedical Investigations (OBI)."
+contact:  batchelorc [at] rsc [.] org
+creators: 
+  - Royal Society of Chemistry
+license: https://creativecommons.org/licenses/by/4.0/
+
+## general ODK Settings ##
+
+repo: rsc-cmo
+git_main_branch: master
+github_org: rsc-ontologies
+ci:
+  - github_actions
+documentation:
+  documentation_system: mkdocs
+export_formats:
+  - owl
+  - obo
```

Comment:
i would recommend also adding a json release, as this is becoming more commonly used now

Reply:
Added in the config & Makefile in [0318010](https://github.com/rsc-ontologies/rsc-cmo/pull/31/commits/0318010c433a86736af085cc68042950f63ca5ad). The actual release files will be build when I rerun `make prepare_release`